src/server/editorServices.ts

```diff
@@ -113,6 +113,7 @@ namespace ts.server {
             this.moduleResolutionHost = {
                 fileExists: fileName => this.fileExists(fileName),
                 readFile: fileName => this.host.readFile(fileName),
+                getCurrentDirectory: () => this.host.getCurrentDirectory(),

```

Comment:
this is the tsserver install location, correct? do not see how this can be useful.